lib/pymedphys/_dicom/header.py

```diff
@@ -1,17 +1,18 @@
-# Copyright (C) 2021 Matthew Jennings
-# Copyright (C) 2019 Cancer Care Associates
-
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-
-#     http://www.apache.org/licenses/LICENSE-2.0
-
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+"""
+Copyright (C) 2021 Matthew Jennings
+Copyright (C) 2019 Cancer Care Associates
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
```

Comment:
Pretty sure we don't want the license header as the module docstring. That's reserved for explaining what this module is for.

Reply:
Fair enough. To be honest, that was pretty mindless change.